pytplot/tplot_math/clip.py

```diff
@@ -46,5 +44,8 @@ def clip(tvar1,ymin,ymax,newtvar):
         new_df = new_df + [tv2_col]
     new_df = np.transpose((list(new_df)))
     #store clipped tvar
-    pytplot.store_data(newtvar, data={'x':tvar_orig.data.index,'y':new_df})
+    if (pytplot.data_quants[tvar1].spec_bins is not None) and (pytplot.data_quants[tvar1].spec_bins_time_varying == True):
+        pytplot.store_data(newtvar, data={'x':tvar_orig.data.index,'y':new_df,'v':pytplot.data_quants[tvar1].spec_bins})
```

Comment:
Even if not time varying, we should still read it in.  This applies to all functions.